### PR TITLE
Adds backgrounds to songs.json for testing

### PR DESCRIPTION
**Pull request by Unesdala**

### CODE REVIEW

**Comment by Unesdala** [comment]
@JoshuaVSherman Any idea where audio is being initialized? (Right now, background is being set to #playSection audio, and being able to manipulate that with JSX would be a lot easier than trying to find some hacky workaround to change the css someway else.)

**Comment by JoshuaVSherman** [comment]
@Unesdala did check the react-player on github? https://github.com/CookPete/react-player

you could add inline style to the root element?

**Comment by Unesdala** [comment]
@JoshuaVSherman Last bit needed is line 83 in musicUtils being covered. Should be, but the way I extended one of the tests isn't covering it, I guess.

Can you take a look at it?

File: src/components/MusicPlayer/index.tsx [hunk 1]
```diff
@@ -99,14 +100,15 @@ export class MusicPlayer extends Component<MProps, MusicPlayerState> {
     const { player } = this.state;
     return (
       <ReactPlayer
-        style={{ backgroundColor: '#eee', textAlign: 'center' }}
+        style={this.musicUtils.setPlayerStyle(song as any)}
```
Comment (JoshuaVSherman) [inline src/components/MusicPlayer/index.tsx:103]: @Unesdala this should be the Song type here not `any`

File: src/components/MusicPlayer/musicUtils.tsx [hunk 2]
```diff
@@ -61,9 +61,43 @@ function textUnderPlayer(song: any) {
   );
 }
 
+function setPlayerStyle(song) {
```
Comment (JoshuaVSherman) [inline src/components/MusicPlayer/musicUtils.tsx:64]: @Unesdala need to provide type for function parameters and a return type

File: src/redux/mapStoreToProps.ts [hunk 3]
```diff
@@ -1,23 +1,24 @@
 export interface Song {
-  title: string;
```
Comment (JoshuaVSherman) [inline src/redux/mapStoreToProps.ts:2]: @Unesdala this is where you can import the `Song` type to use in the other places

**Pull request merged**